src/content/migrate/5.md

```diff
@@ -103,18 +105,29 @@ Yarn: `yarn add webpack@next -D`
 - If you are using Yarn's PnP and the `pnp-webpack-plugin`, we have good news: it is supported by default now. You have to remove it from the configuration.
 - If you are using `IgnorePlugin` with a regular expression as argument, it takes an `options` object now: `new IgnorePlugin({ resourceRegExp: /regExp/ })`.
 
-If you were using WASM via import you should follow this two step process:
+If you were using WebAssembly via import you should follow this two step process:
 
 - Enable the deprecated spec by setting `experiments.syncWebAssembly: true`, to get the same behavior as in webpack 4.
 - After successful migration to webpack 5, change the `experiments` value to `experiments: { asyncWebAssembly: true }` to use the up-to-date spec for WASM integration.
 
 Reconsider `optimization.splitChunks`:
 
 - It's recommended to use either the defaults or `optimization.splitChunks: { chunks: 'all' }`.
-- When using HTTP/2 and long term caching, set `optimization.splitChunks: { chunks: 'all', maxInitialRequests: 30, maxAsyncRequests: 30, maxSize: 100000 }`.
 - When using a custom configuration, replace `name` with `idHint`.
 - It was possible to turn off the defaults by setting `optimization.splitChunks: { default: false, vendors: false }`. We don't recommend doing this, but if you really want to get the same effect in webpack 5: `optimization.splitChunks: { default: false, defaultVendors: false }`.
 
+Consider removing defaults:
+
+- Using `entry: "./src/index.js`: you can omit it, that's the default.
+- Using `output.path: path.resolve(__dirname, "dist")`: you can omit it, that's the default.
+- Using `output.filename: "[name].js"`: you can omit it, that's the default.
+
+Need to support an older browser?
+
+- By default webpack will use your browserslist config to decide which code style to emit.
+- Without a browserslists it would emit ES6 style. You can use `target: ["web", "es5"]` to change it to ES5.
```

Comment:
it was `browserslist` on the previous line, need to be consistent
```suggestion
- Without a browserslist it would emit ES6 style. You can use `target: ["web", "es5"]` to change it to ES5.
```